packages/node/src/methods/app-instance/get-free-balance/controller.ts

```diff
@@ -36,15 +31,14 @@ export default class GetFreeBalanceController extends NodeController {
 
     const stateChannel = await store.getStateChannel(multisigAddress);
 
-    const freeBalanceState = deserializeFreeBalanceState(stateChannel
-      .freeBalance.state as FreeBalanceStateJSON);
+    const ret = stateChannel
```

Comment:
Can we please maintain concrete, specific variable names? `ret` feels a bit too generic, IMHO.

Reply:
Sorry, just saw this was in WIP, didn't mean to early-review :sweat_smile:

Reply:
What do you suggest the name be? I use `ret` if the string `const ret =` can be replaced with `return` (except for the error handling after)